tiny_dnn/activations/softplus_layer.h

```diff
@@ -0,0 +1,51 @@
+/*
+    Copyright (c) 2017, Taiga Nomi
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
+#pragma once
+#include "tiny_dnn/activations/activation_layer.h"
+#include "tiny_dnn/layers/layer.h"
+
+namespace tiny_dnn {
+
+class softplus_layer : public activation_layer {
```

Comment:
@nishnik checkout Pytorch implementation
http://pytorch.org/docs/nn.html?highlight=softplus#torch.nn.Softplus

they add `beta` and `threshold` parameters